routes/user.playlist.js

```diff
@@ -254,28 +300,63 @@ router.route('/:playlist_name/export3')
                         type: 'solid',
                         color: '000000'
                     };
-                    for (var i = 0; i < songObj.minLine; i++) {
+                    var titleMargin = 175
+                    songObj.songs.forEach((song, i) => {
```

Comment:
Can you add some comments explaining what the slides you're trying to make are like?

For example: "create a slide with a black background, white text where the text is centered"